test/index.coffee

```diff
@@ -55,30 +133,33 @@ describe 'Garbage collection', ->
 		.then ->
 			Promise.map IMAGES, (image) ->
 				promiseToBool(docker.getImage(image).inspect())
-		.then (imgs) ->
-			if not _.isEqual(imgs, [false, true, true])
-				throw new Error('Incorrect images removed!')
+		.then (images_found) ->
+			expect(images_found).to.deep.equal([false, true, true])
 
 	it 'should remove more than one image if necessary', ->
 		this.timeout(600000)
 		return Promise.resolve() if SKIP_GC_TEST
 
 		docker = @docker
 
-		# Get the size of the first image, so we can add one to it to remove
-		# the next one in addition
 		Promise.each IMAGES, (image) ->
 			pullAsync(docker, image)
+# First inspect the images we expect to be removed to make more likely that event times are recorded
```

Comment:
Can you fix the indentation of this please

Reply:
Sure, it didn't appear in the diff :?